src/main/java/teammates/common/util/Const.java

```diff
@@ -193,6 +173,9 @@ private Const() {
         public static final String QUERY_LOGS_VERSION = "version";
         public static final String QUERY_LOGS_EXTRA_FILTERS = "extrafilters";
         public static final String QUERY_LOGS_ORDER = "order";
+
+        public static final String NOTIFICATION_TARGET_USER = "usertype";
+        public static final String NOTIFICATION_IS_FETCHING_ALL = "isfetchingall";
```

Comment:
Document uses camel case. Would you like to make a note at the document regarding the change?

Reply:
Okay sure, let me edit the document for this. Thanks for the note!